rackcity/models/pdu_port.py

```diff
@@ -0,0 +1,32 @@
+from django.db import models
+from .asset import Rack
+from django.core.validators import MaxValueValidator, MinValueValidator
+
+
+class PDUPort(models.Model):
+    rack = models.ForeignKey(
+        Rack,
+        on_delete=models.CASCADE,
+        verbose_name="rack",
+    )
+    LEFT_RIGHT_CHOICES = [
```

Comment:
What is this for?

Reply:
The choices field in Django makes you give is a tuple of what's stored in the database and what you display (think it's a legacy thing) - in our case both are the same